src/components/VAudioDetails/VAudioDetails.vue

```diff
@@ -104,6 +104,18 @@ export default defineComponent({
       required: true,
     },
   },
+  setup(props) {
+    const audioFormats = computed(() => {
+      if (!props.audio.alt_files) return props.audio.filetype
+      const altFormats = props.audio.alt_files.map(
+        (altFile) => altFile.filetype
+      )
+      altFormats.unshift(props.audio.filetype)
```

Comment:
Judging from the code here, it seems that we should not have the duplicated file types, but we do: http://localhost:8443/audio/69368299-c262-4212-b877-2e6b60b4f473 (note that the format is set as `MP3, MP3`). These are different bitrates, but they are still both MP3.
Maybe we should use a set here to prevent duplicate types?